v2/community/updating-supertokens.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: updating-supertokens
+title: Updating Supertokens SDKs and Core
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./community/updating-supertokens.mdx -->
+
+# Updating Supertokens SDKs and Core
+
+Not all older releases of every supertokens Core, front-end SDK, and back-end SDK are compatible with each other. In order to maintain compatability
+and keep up on new features, we recommend updating and upgrading all of your SDKs regularly, starting with the supertokens core, then the back-end SDK, then the front-end SDK.
+
+The following is a link to each core and SDK changelog, containing steps to migrate from each version. You can use our [compatability table](../../community/compatibility-table) to see which versions you need to upgrade to.
+
+## Core Migration
+
+- ### Core
+[supertokens-core](https://github.com/supertokens/supertokens-core/blob/master/CHANGELOG.md)
```

Comment:
you should say if using managed service, please email us on team@supertokens.com from your registered email ID requesting a core update. For seld hosted, see the changelog